tests/unit/test_session.py

```diff
@@ -1696,6 +1696,191 @@ def unit_of_work(txn, *args, **kw):
             ],
         )
 
+    def test_run_in_transaction_w_exclude_txn_from_change_streams(self):
```

Comment:
Can you also add a test for ILB. use case, when there is no begin transaction and "ExecuteSql" should have `exclude_txn_from_change_streams` set

Reply:
Added